src/Workspaces/VisualBasic/Portable/Classification/Worker.vb

```diff
@@ -129,14 +129,60 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Classification
             ElseIf trivia.Kind = SyntaxKind.CommentTrivia Then
                 AddClassification(trivia, ClassificationTypeNames.Comment)
             ElseIf trivia.Kind = SyntaxKind.DisabledTextTrivia Then
-                AddClassification(trivia, ClassificationTypeNames.ExcludedCode)
+                ClassifyDisabledText(trivia, triviaList)
             ElseIf trivia.Kind = SyntaxKind.ColonTrivia Then
                 AddClassification(trivia, ClassificationTypeNames.Punctuation)
             ElseIf trivia.Kind = SyntaxKind.LineContinuationTrivia Then
                 AddClassification(New TextSpan(trivia.SpanStart, 1), ClassificationTypeNames.Punctuation)
+            ElseIf trivia.Kind = SyntaxKind.ConflictMarkerTrivia Then
+                ClassifyConflictMarker(trivia)
+            End If
+        End Sub
+
+        Private Sub ClassifyConflictMarker(trivia As SyntaxTrivia)
+            AddClassification(trivia, ClassificationTypeNames.Comment)
+        End Sub
+
+        Private Sub ClassifyDisabledText(trivia As SyntaxTrivia, triviaList As SyntaxTriviaList)
+            Dim index = triviaList.IndexOf(trivia)
+            If index >= 2 AndAlso
```

Comment:
Possibly now.  It used to be not possible due to the different syntax models.  Now that we've unified, tihs area is ripe for an overhaul.  I'll add to my cleanup backlog.